webapp/src/routes/app/your-cars/+page.svelte

```diff
@@ -148,39 +138,45 @@
     }
 </script>
 
-{#if errorMessage}
-    <InlineNotification title="Error:" bind:subtitle={errorMessage} />
-{/if}
-
 <div class="button-container" style="">
-    <Button style="margin: 1rem;" icon={Add} on:click={() => (isAddModalOpen = true)}
-        >New Car</Button
+    <Button
+        aria-label="new-car-button"
+        style="margin: 1rem;"
+        icon={Add}
+        on:click={() => (isAddModalOpen = true)}>New Car</Button
     >
 </div>
 
-<div bind:this={list} style="overflow: auto">
+<div>
     {#key data.cars}
         {#each data?.cars as car}
             <!-- svelte-ignore a11y-click-events-have-key-events a11y-no-static-element-interactions -->
-            <div on:click={() => selectCarIndex(car.id, car.details)}>
+            <div id={car.id} on:click={() => selectCarIndex(car.id, car.details)}>
                 <CarDisplay car={car.details}></CarDisplay>
             </div>
         {/each}
     {/key}
-
+</div>
+<IntersectionObserver element={loadTrigger} bind:intersecting>
```

Comment:
intersect event won't work in this case. From the documentation, intersect only trigger when state change so f the first load doesn't cause the page to overflow, there is no way for us to load more stuff because the state doesn't change

Reply:
it does trigger the first time but if after that, the page is not long enough to scroll yet then the loadTrigger remains on the screen. intersect then doesn't trigger anymore because there is nothing that can cause it to change state